components/auth/signIn.tsx

```
@@ -1,139 +1,140 @@
+							"Login"
+						)}
+					</button>
+
+					<div className="mt-4 space-y-2 sm:mt-0 text-sm">
+						<p className="text-foreground">
+							Don&apos;t have an account?{" "}
+							<a
+								href="#"
+								onClick={() => stateSync("SIGNUP")}
+								className={`underline text-foreground`}
+							>
```

Biome (2.1.2)

[error] Use a button element instead of an a element.

Anchor elements should only be used for default sections or page navigation
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)